### ISSUE TITLE
Filer work day

### ISSUE THREAD

**Issue by modeswitch** [filerjs/filer#363]
Any interest in having an in-person work day for Filer? The idea is we pick one or two new features and implement them. Whatever design work is required should be done ahead of time so that the entire day can be devoted to pair programming. Qorum would be 3-4 people.

@humphd @sedge @alicoding @gideonthomas

**Comment by sedge** [comment]
I'm always up for a hack fest! Dev on thimble is pretty all consuming at the moment though...

**Comment by humphd** [comment]
I'm interested, but not until the fall. We're slammed getting Thimble/Bramble shipped. Filer has been holding up great being used there. It's been a few months since we found a bug. We even used symlinks, sh.find, and some other advanced stuff and it just works :) We wrote some good code.

On Aug 12, 2015 10:21 AM, Kieran Sedgwick <notifications@github.com> wrote:

I'm always up for a hack fest! Dev on thimble is pretty all consuming at the moment though...
